src/services/toggle-service.test.js

```diff
@@ -0,0 +1,58 @@
+var dom = require('@nymag/dom');
+
+// Mock DOM elements
+const mockSpaceElements = [
+  {getAttribute: ()=>'/_components/clay-space/instances/cjfb7nklu000w3i5xzvizb72e'},
+  {getAttribute: ()=>'/_components/clay-space/instances/cjfjy3tdh000p3i5zg3gaubli'},
+  {getAttribute: ()=>'/_components/clay-space/instances/cjgpep0we000v3i5zgolixf2u'},
+];
+
+import * as lib from './toggle-service.js';
+import { VuexStoreStub } from '../../test/utils.js';
+
+describe('toggle-service', ()=>{
+  let storeStub, dispatchCalls;
+
+  function mockDependencies() {
+    dom = jest.fn();
+    dom.find = jest.fn();
+  };
+
+  beforeEach(()=>{
+    mockDependencies();
+    storeStub = new VuexStoreStub();
+    dispatchCalls = [];
+  });
+
+  // TODO: these tests are WIP
```

Comment:
are these going to be a part of this PR, or a separate one?

Reply:
going to be a separate one